### ISSUE TITLE
Use nginx proxy together with wordpress

### ISSUE THREAD

**Issue by hristod** [evertramos/docker-compose-letsencrypt-nginx-proxy-companion#97]
Hello!

First, thank you for this great repo! This is exactly what I need, together with `docker-wordpress-letsencrypt`!

Is there something I can read about how to configure this repo to work together with your wordpress image: https://github.com/evertramos/docker-wordpress-letsencrypt? I'm a bit new to docker, and need some help to get started...

**Comment by hristod** [comment]
One thing I don't get is what do I put here:
` NGINX_FILES_PATH=/path/to/your/nginx/data`

**Comment by evertramos** [comment]
Hello @hristod,

Thanks for using our scripts!

First of all, you could start by setting up the web proxy, before starting the wordpress... follow the readme on the repo https://github.com/evertramos/docker-compose-letsencrypt-nginx-proxy-companion and you should be just fine with it.

The NGINX file path, is where you want to keep your nginx files, which are config files, ssl certificates, htpasswd files etc. Just create a folder and set to this variable and check the folder again after start the proxy with `$./start.sh`.

Please let me know how it goes.

Than you try the wordpress including the same webproxy network.

P.S.: Make sure you don´t have any service listening on port 80 and 443 when starting the proxy.

**Comment by hristod** [comment]
Hey, thanks so much for this! What I've done is, I've first ran the wordpress container, which creates the "data" folder, with all the WP files needed. Then I put the path to the `data` folder as the value of `NGINX_FILES_PATH` and run the proxy. It worked like a charm!

One side question, if you have time to reply: I'm looking on running this on a raspberry pi 3, arm64 architecture. Have you ever tested it on such a device?

**Comment by evertramos** [comment]
@hristod... what you did is not right... sorry to say that... please clean everything up and try again as I mentioned you will be much happier. 

I do ran on rpi some services, and connect to server using overlay network, but never ran an entire proxy and service on rpi. Let us know if you get through it.

**Comment by hristod** [comment]
I did try what you mentioned and it didn't work :( What I did exactly is:
1) Cloned proxy repo
2) edited edited .env file and set the following: `NGINX_FILES_PATH=/path/to/your/nginx/data`
3) ./start.sh
The container started
4) pulled the wordpress container repo
5) copied .env.sample to .env, didn't change anything there
6) docker-compose up -d

When I go to my localhost, I get the following: "503 Service Temporarily Unavailable", on both http and https.

P.S.: I've created the `webproxy` network before. I get no errors when running the containers.
